ui/src/journals/containers/DetailPageContainer.tsx

```diff
@@ -71,10 +73,7 @@ export const DetailPage = ({ result, isCatalogerLoggedIn }: { result: Journal, i
                 <PageHeader
                   className="site-page-header"
                   title={shortTitle}
-                  subTitle={
-                    // @ts-ignore
-                    publisher ? `(${publisher.toArray().map((p: string) => p)})` : false
-                  }
+                  subTitle={publisher && `(${publisher.toArray().map((p: string) => p)})`}
```

Comment:
are we sure it won't fail with publisher `undefined`?

Reply:
This `publisher &&` prevents from executing if undefined the same as ternary operator `publisher ?` (checked and if publisher is undefined it just does not render subTitle)